content/scripts/matches-in-text.js

```diff
@@ -0,0 +1,18 @@
+function* findMatchesInText(needle, haystack, { inQuotes = false } = {}) {
```

Comment:
I thought this little function was kinda neat so I blogged about it: https://www.peterbe.com/plog/findmatchesintext